boxoffice/views/admin.py

```diff
@@ -24,11 +25,26 @@ def index():
     return dict(user=g.user)
 
 
+def jsonify_ic(ic):
+    if ic:
+        return {
+            'id': ic.id,
+            'name': ic.name,
+            'title': ic.title,
+            'url': url_for('admin_item_collection', ic_id=ic.id),
```

Comment:
Use `self.url_for('admin_item_collection')`. If the `url_for` method doesn't recognise that action, add it.